src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/services/NomisApiService.kt

```diff
@@ -95,6 +96,17 @@ class NomisApiService(@Qualifier("nomisApiWebClient") private val webClient: Web
       .bodyToMono(CreateSentencingAdjustmentResponse::class.java)
       .awaitSingle()
 
+  suspend fun updateSentenceAdjustment(
+    adjustmentId: Long,
+    request: UpdateSentencingAdjustmentRequest
+  ): Unit? =
```

Comment:
`Unit?` looks wrong

Reply:
Yep. Was due to to awaitFirst which didn't work (there is no body), so did awaitFirstOrNull which forced a nullable Unit

I'll take a second look